apps/explorer/src/pages/validators/Validators.tsx

```diff
@@ -247,7 +245,9 @@ function ValidatorPageResult() {
             return null;
         const apys = Object.values(rollingAverageApys);
```

Comment:
can we just filter out the zero APY validators here ? 
```suggestion
        const apys = Object.values(rollingAverageApys).filter( ... );
```